src/modules/launcher/Wox.Plugin/SharedCommands/DefaultBrowserInfo.cs

```diff
@@ -0,0 +1,235 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO.Abstractions;
+using System.Text;
+using Wox.Plugin.Logger;
+using Wox.Plugin.SharedCommands.Win32;
+
+namespace Wox.Plugin.SharedCommands
+{
+    /// <summary>
+    /// Contains information (e.g. path to executable, name...) about the default browser.
+    /// </summary>
+    public static class DefaultBrowserInfo
+    {
+        private static readonly object _updateLock = new object();
+        private static int _lastUpdateTickCount = -1;
+
+        public static readonly string MSEdgePath = Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86) + @"\Microsoft\Edge\Application\msedge.exe";
```

Comment:
What happens if edge isn't actually there? Will the Plugin crash?

Reply:
No. Then we get an error message that the file path can't be found. There is an message box that we show if shellOpen failed.

These PR fixes the crash that happend if we can't detect the default browser and the parameters of shellOpen are null then. With the new code we then fall back to edge.

Reply:
Looks good to me, then. THank you!